caravel/model/affiliation.py

```diff
@@ -51,9 +51,10 @@ class AffiliationMixin(ndb.Model):
     _aff = None
 
     AFFILIATIONS = [
-        ("bsd_hospital", "BSD/Hospital Staff"),
+        ("bsd_hospital", "BSD/Hospital"),
+        ("bfi_booth", "BFI/Booth"),
         ("osha_faculty_staff", "University Faculty/Staff"),
```

Comment:
Or Just (Non-BSD) Faculty/Staff.

My point is to try to make the two sets as disjoint as possible. University is a super set of BSD/Hospital.